### PR TITLE
adding works

### PR DESCRIPTION
**Pull request by OlafKozak**
apuD

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: fibonacci
🏅 Granted 6 XP for @OlafKozak

**Comment by ziobron** [comment]
⏱ 3 XP extra for @OlafKozak for on-time delivery

File: homework/fibonacci/fibonacci.hpp [hunk 1]
```diff
@@ -1,11 +1,28 @@
...
-    // TODO: Your implementation goes here
-    return 0;
+    int first = 0;
+    int second = 1;
+    if(!sequence){
+        return 0;
+    }
+    if(sequence == 1){
+        return 1;
+    }
```
Comment (nauka-programowania-MA) [inline homework/fibonacci/fibonacci.hpp:11]: Sequence hold exactly the same value as you want to return for 0 and 1 :)

Better write:

`if (sequence < 2) {
    return sequence;
}`

File: homework/fibonacci/fibonacci.hpp [hunk 2]
```diff
@@ -1,11 +1,28 @@
...
+    for(int i=1;i<sequence;i++){
```
Comment (nauka-programowania-MA) [inline homework/fibonacci/fibonacci.hpp:12]: Add space between operations like '+' '-' '!=' '==' '%' etc...

File: homework/fibonacci/fibonacci.hpp [hunk 3]
```diff
@@ -1,11 +1,28 @@
...
 
 int fibonacci_recursive(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+    if(!sequence){
+        return 0;
+    }
+    if(sequence == 1){
+        return 1;
+    }
```
Comment (nauka-programowania-MA) [inline homework/fibonacci/fibonacci.hpp:26]: Sequence hold exactly the same value as you want to return for 0 and 1 :)

Better write:

`if (sequence < 2) {
    return sequence;
}`

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**